src/utilities.jl

```diff
@@ -13,23 +13,71 @@ Returns the number of samples present in the span `aligned`.
 n_samples(aligned::AlignedSpan) = length(indices(aligned))
 
 """
-    consecutive_subspans(span::AlignedSpan, duration::Period)
+    consecutive_subspans(span::AlignedSpan, duration::Period; keep_last=true)
+    consecutive_subspans(span::AlignedSpan, n::Int; keep_last=true)
 
 Creates an iterator of `AlignedSpan` such that each `AlignedSpan` has consecutive indices
-which cover all of the original `span`'s indices. In particular,
+which cover the original `span`'s indices (when `keep_last=true`).
```

Comment:
```suggestion
which cover the original `span`'s indices (when `keep_last=true`). In particular,
```
(or update the funky bullets below so that they are not bullets OR have some other intro!)